great_expectations/datasource/fluent/sqlite_datasource.py

```diff
@@ -183,10 +184,13 @@ def add_table_asset(
         table_name: str,
         schema_name: Optional[str] = None,
         order_by: Optional[SortersDefinition] = None,
+        batch_metadata: Optional[BatchMetadata] = None,
     ) -> SqliteTableAsset:
         return cast(
             SqliteTableAsset,
-            super().add_table_asset(name, table_name, schema_name, order_by),
+            super().add_table_asset(
+                name, table_name, schema_name, order_by, batch_metadata
+            ),
```

Comment:
I would prefer that we pass the keywords here. Someone could re-order the args in the superclass and mess this up. Also, in `PandasDatasource` we actually constrain the positional args to match pandas (via `*`) and then we require the rest as kwargs (see: [stub files](https://github.com/great-expectations/great_expectations/blob/92a7ebcbcecc78bcf9d2dcd6f2ffe24e51669c98/great_expectations/datasource/fluent/pandas_datasource.pyi#L169)). That means this is inconsistent to a small degree.